.github/workflows/docs-build.yml

```diff
@@ -78,24 +80,21 @@ jobs:
       - name: Test Examples in Docs
         if: ${{ matrix.check == 'doctest' }}
         working-directory: ./docs/source-${{ matrix.pkg-name }}
-        env:
-          FAST_DOCS_DEV: 1
         run: |
           make doctest
           make coverage
 
       - name: Check External Links
         if: ${{ matrix.check == 'linkcheck' }}
         working-directory: ./docs/source-${{ matrix.pkg-name }}
-        env:
-          FAST_DOCS_DEV: 1
         run: make linkcheck SPHINXOPTS="--keep-going"
 
+      - name: Full build for deployment
+        if: github.event_name == 'push'
+        run: echo "FAST_DOCS_DEV=0" >> $GITHUB_ENV
```

Comment:
Will this override the `FAST_DOCS_DEV` value for all the subsequent steps? Could this be a problem? Should you reset it after?

Reply:
The HTML is the last option so it shall be fine...